### ISSUE TITLE
badfuncs_driver: Assertion `after_symbols != NULL' failed

### ISSUE THREAD

**Issue by msrb** [rpminspect/rpminspect#915]
This issue was reported downstream:

```
$ rpm -q rpminspect
rpminspect-1.11-0.1.202210181602git0574962.fc37.x86_64

$ rpminspect-redhat virt-top-1.1.1-6.el9
LibClamAV Warning: **************************************************
LibClamAV Warning: ***  The virus database is older than 7 days!  ***
LibClamAV Warning: ***   Please update it as soon as possible.    ***
LibClamAV Warning: **************************************************
rpminspect: ../lib/inspect_badfuncs.c:142: badfuncs_driver: Assertion `after_symbols != NULL' failed.
/usr/bin/rpminspect-redhat: line 2: 103996 Aborted                 (core dumped) /usr/bin/rpminspect -c /usr/share/rpminspect/redhat.yaml $*

```

**Comment by rwmjones** [comment]
Note it's an OCaml program.

**Comment by msrb** [comment]
`ppc64-diag-2.7.9-1.el9` seems to be affected as well.

**Comment by dcantrell** [comment]
This is fallout from the list_add() refactoring patch I pushed yesterday.  Forgot one to drop one unnecessary assert() call.  Apologies.  Patch coming here in seconds.
